guides/01_getting-started/03_sharing-your-app.md

```diff
@@ -451,4 +451,16 @@ Gradio DOES NOT ALLOW access to:
 
 - **Any other paths on the host machine**. Users should NOT be able to access other arbitrary paths on the host.
 
+Sharing your Gradio application will also allow users to upload files to your computer or server. You can set a maximum file size for uploads to prevent abuse and to preserve disk space. You can do this with the `max_file_size` parameter of `.launch`. For example, the following two code snippets limit file sizes to 5 megabytes.
```

Comment:
Nit: maybe we should clarify its 5 mb per file
```suggestion
Sharing your Gradio application will also allow users to upload files to your computer or server. You can set a maximum file size for uploads to prevent abuse and to preserve disk space. You can do this with the `max_file_size` parameter of `.launch`. For example, the following two code snippets limit file sizes to 5 megabytes per file.
```